modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/resources/content/Language.properties

```diff
@@ -1,3 +1,4 @@
 applications-menu=Applications Menu
 javax.portlet.title.com_liferay_product_navigation_applications_menu_web_internal_portlet_ProductNavigationApplicationsMenuPortlet=Applications Menu
-recently-visited=Recently Visited
\ No newline at end of file
+recently-visited=Recently Visited
+open-menu=Open menu
```

Comment:
Run `gradlew buildLang -a` to generate the keys for all languages

Reply:
Yes cap

Reply:
858b6de